### ISSUE TITLE
Error occurred while evaluating on validation set

### ISSUE THREAD

**Issue by Fingolfin007** [charlesq34/frustum-pointnets#47]
When trying to evaluate based on pre-trained snapshot by calling
```
CUDA_VISIBLE_DEVICES=0 sh scripts/command_test_v1.sh
```

An error occurred: ```number of files for evaluation: 3769
ERROR: Couldn't read: 000560.txt of ground truth.```

**Comment by TRON15** [comment]
Hi, I guess you didn't prepare the training labels of KITTI，look at the script command_test_v1.sh carefully.
`train/kitti_eval/evaluate_object_3d_offline dataset/KITTI/object/training/label_2/ train/detection_results_v1`
you need to download the "Download training labels of object data set (5 MB)" from [KITTI](http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=3d) and organize the folders as [dateset/README.md](https://github.com/charlesq34/frustum-pointnets/tree/master/dataset) requests. In fact, you only need to place the label_2 on the right path to run this scripts.
I hope this can help you.
